lib/dpul_collections/indexing_pipeline/figgy/hydration_consumer.ex

```diff
@@ -88,6 +88,43 @@ defmodule DpulCollections.IndexingPipeline.Figgy.HydrationConsumer do
     |> Broadway.Message.put_data(message_map)
   end
 
+  def handle_message(
+        _processor,
+        message = %Broadway.Message{
+          data: %{
+            internal_resource: internal_resource,
+            metadata: %{
+              "resource_id" => [%{"id" => resource_id}],
+              "resource_type" => [resource_type]
+            }
+          }
+        },
+        %{cache_version: cache_version}
+      )
+      when internal_resource in ["DeletionMarker"] and
+             resource_type in ["EphemeraFolder", "EphemeraTerm"] do
+    # Only process messages where the deleted resource has an exisiting
+    # hydration cache entry. If one does not exist, it means that the resource
+    # has not been indexed into DPUL-C.
+    resource = IndexingPipeline.get_hydration_cache_entry!(resource_id, cache_version)
```

Comment:
Should we be passing this cache_version? I think this is the `cache_version` for the DeletionMarker, no?

Reply:
I got cache_version confused with cache_order.